pkg/apis/crd/v1alpha1/types.go

```diff
@@ -291,3 +291,151 @@ type TLSProtocol struct {
 	// SNI (Server Name Indication) indicates the server domain name in the TLS/SSL hello message.
 	SNI string `json:"sni,omitempty"`
 }
+
+// Source describes the source spec of the traceflow.
```

Comment:
The references to "traceflow" (3 in this file) are probably not correct. It seems that eventually we may want to use the same structs for the `Traceflow` API and the `PacketCapture` API, but for now at least they don't share an API version and we should remove the references to "traceflow" here.